.buildkite/gpu_pipeline/pipeline.yml

```diff
@@ -200,6 +218,21 @@ steps:
           slurm_time: 8:00:00
           slurm_exclusive:
 
+      - wait
+
+      - label: "gpu_aquaplanet_dyamond - weak scaling plots"
+        command:
+          - mkdir -p gpu_aquaplanet_dyamond_ws
+          - >
+            julia --color=yes --project=examples post_processing/plot_gpu_strong_scaling.jl gpu_aquaplanet_dyamond_ws
```

Comment:
The filename here is a bit misleading (?) maybe we generalize the `.jl` filename then it can still pick up the `gpu_aquaplanet_dyamond_ws` or `gpu_aquaplanet_dyamond_ss` argument cc @nefrathenrici

Reply:
Ah it's a typo and should simply be `plot_gpu_weak_scaling`